admin/src/hooks/usePreviewButton.ts

```diff
@@ -14,17 +15,15 @@ export interface UsePreviewButtonReturn {
 }
 
 const usePreviewButton = (
-  layout: any,
+  uid: UID.ContentType | undefined,
```

Comment:
Changed from `layout` to `uid` because we only need `uid`.

`uid` is undefined when the document has not been created yer, which is why we no longer need `isCreating`.